Tests/test_PDB.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 by Simon Duerr.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""Tests for PDB module."""
+
+import gzip
+import sys
+import unittest
+import warnings
+
+from Bio import PDB
+
+from Bio.PDB import PDBParser, MMCIFParser
+
+
+class PDBTestBaseClass(unittest.TestCase):
+    """Base class for Bio.PDB unit tests."""
+
+    def setUp(self):
+        self.PDBParser = PDBParser()
+        self.MMCIFParser = MMCIFParser()
+        pass
+
+    def test_PDB_read_pdb_pdbgz(self):
+        """Test read function with supported formats.
+
+        Can read: pdb, pdb.gz, cif, cif.gz, mmtf
+        """
+        structure = PDB.read("PDB/1A8O.pdb", format="pdb")
+        structure_gzip = PDB.read("PDB/1A8O.pdb", format="pdb")
+
+        with gzip.open("PDB/1A8O.pdb.gz", mode="rt") as handle:
+            structure_gzip_handle = PDB.read(handle, format="pdb")
+
+        with open("PDB/1A8O.pdb") as handle:
+            structure_handle = PDB.read(handle, format="pdb")
+
+        structure_parser = self.PDBParser.get_structure("1A8O", "PDB/1A8O.pdb")
+
+        self.assertEqual(structure, structure_gzip)
+        self.assertEqual(structure, structure_gzip_handle)
+        self.assertEqual(structure, structure_handle)
+        self.assertEqual(structure, structure_parser)
+
+    def test_PDB_read_format(self):
+        """Test format errors."""
+        with self.assertRaises(TypeError):
+            _ = structure = PDB.read("PDB/1A8O.pdb", format=0)
+
+        with self.assertRaises(ValueError):
```

Comment:
Remove this to be consistent with the changes proposed elsewhere (to make `format` case-insensitive).